pkg/printers/printers.go

```diff
@@ -91,11 +86,27 @@ func GetStandardPrinter(typer runtime.ObjectTyper, encoder runtime.Encoder, deco
 		if err != nil {
 			return nil, fmt.Errorf("error reading template %s, %v\n", formatArgument, err)
 		}
-		jsonpathPrinter, err := NewJSONPathPrinter(string(data))
+
+		formatArgument = string(data)
+		fallthrough
+	case "jsonpath":
```

Comment:
Combined this case with the `template` and `go-template` case.  You can use your combined printer code, right?